synfig-studio/src/gui/instance.cpp

```diff
@@ -223,25 +223,128 @@ Instance::set_redo_status(bool x)
 	signal_undo_redo_status_changed()();
 }
 
+bool
+studio::Instance::run_plugin_with_arguments(std::string plugin_path, const std::vector<std::string>& args)
+{
+	OneMoment one_moment;
+
+	bool result = false;
+	String output;
+	String command = "";
+
+	// Path to python binary can be overridden
+	// with SYNFIG_PYTHON_BINARY env variable:
+	const char* custom_python_binary=getenv("SYNFIG_PYTHON_BINARY");
+	if(custom_python_binary) {
+		command=custom_python_binary;
+		if (!App::check_python_version(command)) {
+			output=_("Error: You need to have Python 3 installed.");
+			command="";
+		}
+	} else {
+	// Set path to python binary depending on the os type.
+	// For Windows case Python binary is expected
+	// at INSTALL_PREFIX/python/python.exe
+		std::list< String > binary_choices;
+		binary_choices.push_back("python");
+		binary_choices.push_back("python3");
+		std::list< String >::iterator iter;
+		for(iter=binary_choices.begin();iter!=binary_choices.end();iter++)
+		{
+			String python_path;
+#ifdef _WIN32
+			python_path = "\"" + App::get_base_path()+ETL_DIRECTORY_SEPARATOR+"python"+ETL_DIRECTORY_SEPARATOR+*iter+".exe" + "\"";
```

Comment:
I know there's a lot of places that uses `ETL_DIRECTORY_SEPARATOR`, but I do believe it is better for readability to just use "/" instead: both Win, Mac & Linux accepts it as separator.

Reply:
It's code that was already there, this kind of changes are beyond the scope of this merge request IMO.

Reply:
Ok.